packages/jaspr/lib/src/foundation/styles/properties/filter.dart

```diff
@@ -0,0 +1,188 @@
+import 'angle.dart';
+import 'color.dart';
+import 'unit.dart';
+
+typedef BackdropFilter = Filter;
+
+class Filter {
+  const Filter._(this.value);
+
+  /// The css value
+  final String value;
+
+  // Keyword value(s)
+  static const Filter none = Filter._('none');
+
+  // Global values
+  static const Filter inherit = Filter._('inherit');
+  static const Filter initial = Filter._('initial');
+  static const Filter revert = Filter._('revert');
+  static const Filter revertLayer = Filter._('revert-layer');
+  static const Filter unset = Filter._('unset');
+
+  const factory Filter(List<FilterFn> filterValues) = _Filter;
```

Comment:
Make this named as `Filter.all` or `.list` so its more explicit that this combines multiple filters.